geomesa-tools/bin/common-functions.sh

```diff
@@ -101,19 +101,59 @@ function containsElement() {
 function registerAutocomplete() {
   eval compFile="~/.bash_completion" # resolve tilde
   [[ -f ${compFile} ]] || touch ${compFile}
-  # Search .bash_completion for this entry so we don't add it twice
-  head=$(head -n 1 ${GEOMESA_CONF_DIR}/autocomplete.sh)
-  res=$(grep -F $head ${compFile})
-  if [[ -z "${res}" ]]; then
-    echo "Installing Autocomplete Function"
-    cat ${GEOMESA_CONF_DIR}/autocomplete.sh >> ${compFile}
-    echo "Autocomplete function available, to use now run:"
-    echo ". ${compFile}"
+  if [[ -f "${GEOMESA_HOME}/conf/autocomplete.sh" ]]; then
+    head=$(head -n 1 ${GEOMESA_HOME}/conf/autocomplete.sh) # Don't use GEOMESA_CONF_DIR as it may not be properly set at this point
+    # Search .bash_completion for this entry so we don't add it twice
+    res=$(grep -F $head ${compFile})
+    if [[ -z "${res}" ]]; then
+      echo "Installing Autocomplete Function"
+      cat ${GEOMESA_HOME}/conf/autocomplete.sh >> ${compFile}
+      echo "Autocomplete function available, to use now run:"
+      echo ". ${compFile}"
+    fi
   fi
 }
 
-echo >&2 "Using %%gmtools.dist.name%%_HOME = $%%gmtools.dist.name%%_HOME"
+# Version check
+GEOMESA_TOOLS_VERSION=%%gmtools.version%%
```

Comment:
are these version checks necessary? seems unlikely someone would mix-and-match...